### ISSUE TITLE
Navigate backwards with keys if entered into hierarchy (column view)

### ISSUE THREAD

**Issue by eljefuri** [elementary/files#693]
If _in column view_ I have navigated to (lets say) the Downloads folder via the side-bar, I would like to be able to press the left arrow ← to be able to traverse back down the hierarchy, as I can do in macOS Finder. This was always an extremely common use case for me. As it stands presently, I must mouse to where I want to go next, even if it is a simple single folder down.

Many thanks to the Files developers for continuing to humor my suggestions. :-)

**Comment by peteruithoven** [comment]
Doesn't this already work this way? I only notice that I can't navigate using the arrows keys to and from the sidebar items. 
To clarify, if the Sidebar has focus I can select items in the sidebar using the up / down arrow keys, but I can't open an item using the right arrow key. When navigating through the files I can't press left to select items in the sidebar anymore.

**Comment by eljefuri** [comment]
I updated my comment in the issue to better reflect what I meant. This is not about traversing the sidebar, this is about traversing the whole directory tree. In other words, if I entered into the ~/Downloads folder by clicking on Downloads, I want to be able to press the left arrow key to get to ~ ($HOME). Does this make sense?

**Comment by eljefuri** [comment]
Hi @jeremypw yes they would shift to the right. This is how every version of the macOS Finder that I have used worked, and it was great for rapidly traversing a directory hierarchy. Nice that there is already a kind of shortcut for this. Per #697 how would I have discovered that?

**Comment by eljefuri** [comment]
Makes sense; I am not very familiar with Windows. Alt-Up is very useful, thank you for telling me about that. That said, it would be nice if I could also use the left arrow key when in a folder for what is more or less the same effect, and it would be nice if it were visually represented by the columns moving to the right. Two issues? :-)

**Comment by jeremypw** [comment]
I think in this case the two things are so closely related a single issue is OK - navigate to parent and shift columns to the right on LeftArrow in Column View root.

**Comment by eljefuri** [comment]
@jeremypw Is attaching a bounty to this issue the best way to get it moved out of "Wishlist" and into development?

**Comment by jeremypw** [comment]
@eljefuri : "Wishlist" means that it is considered a new feature rather than a fault with an existing feature. So adding a bounty would not in itself change that - but it would make it more likely that a developer (including third party developers) will be inspired to address it.
